docker-compose.yml

```diff
@@ -95,10 +95,12 @@ services:
       - ":9410"
       - "--mqtt-addr"
       - "mqtt://mqtt:1883"
-      - "--v2g-pem-file"
+      - "--trust-anchor-pem-file"
```

Comment:
As we're being explicit about cso vs mo everywhere else now - perhaps we should be explicit here that these are our mo trust anchors?